src/rules/requireReturnType.js

```diff
@@ -25,6 +25,39 @@ export default (context) => {
         return returnNode.argument === null || returnNode.argument.name === 'undefined' || returnNode.argument.operator === 'void';
     };
 
+    const getIsReturnTypeAnnotationUndefined = (targetNode) => {
+        const isReturnTypeAnnotationLiteralUndefined = _.isMatch(
```

Comment:
I don't like the approach taken here (it is too verbose).

I would prefer this written as:

```js
_.get(targetNode, 'functionNode.returnType.typeAnnotation.id.name', 0) === undefined;
// ...
```